hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -80,10 +73,10 @@ The BOOT_ORDER property defines the sequence for the different boot modes. It is
 * 0x1 - SD CARD  
 * 0x2 - NETWORK  
 * 0x3 - USB device boot - Reserved - Compute Module only.
-* 0x4 - USB mass storage boot (since pieeprom-2020-09-03.bin)
-* 0xf - RESTART (loop) - start again with the first boot order field. (since pieeprom-2020-09-03.bin)
+* 0x4 - USB mass storage boot (since 2020-09-03)
+* 0xf - RESTART (loop) - start again with the first boot order field. (since 2020-09-03)
 
-Default: 0x1  
+Default: 0xf41 (since 2020-09-03 previously 0x1)
```

Comment:
Sorry to nitpick, but I think `0xf41 (0x1 in versions prior to 2020-09-03)` might be slightly clearer? (and also matches the style used elsewhere)

Reply:
not sure it is clearer but updated to consistent style